cinema_service/settings.py

```diff
@@ -121,7 +121,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = False
```

Comment:
Oops, missed it in the initial review. Maybe you meant `USE_TZ = False`? :)

Reply:
😊